files/test_task_helper.py

```diff
@@ -64,7 +64,7 @@ def task(self, args):
                 'kind': 'python.task.helper/exception',
                 'issue_code': 'EXCEPTION',
                 'msg': 'does not work',
-                'details': { 'class': 'Exception' }
+                'details': { 'class': 'Exception', 'debug': [] }
```

Comment:
Would it be worth adding a debug statement in the task?

Reply:
I'm not sure if it's necessary since there's already a test checking that debug statements work correctly.